x-pack/packages/kbn-elastic-assistant-common/impl/schemas/prompts/bulk_crud_prompts_route.schema.yaml

```diff
@@ -5,8 +5,9 @@ info:
 paths:
   /internal/elastic_assistant/prompts/_bulk_action:
```

Comment:
Please change it according to the https://github.com/elastic/kibana/blob/main/x-pack/plugins/elastic_assistant/server/routes/prompts/bulk_actions_route.ts#L112
to `/api/security_ai_assistant/prompts/_bulk_action`